seed/tasks.py

```diff
@@ -322,49 +323,50 @@ def _finish_delete_cycle(cycle_id, prog_key):
 @lock_and_track
 def delete_organization_column(column_pk, org_pk, prog_key=None, chunk_size=100, *args, **kwargs):
     """Deletes an extra_data column from all merged property/taxlot states."""
-
-    column = Column.objects.get(id=column_pk, organization_id=org_pk)
-
     progress_data = ProgressData.from_key(prog_key) if prog_key else ProgressData(
         func_name='delete_organization_column', unique_id=column_pk)
 
+    tasks = []
+    tasks.append(
+        _generate_tasks_to_delete_organization_column.subtask((column_pk, org_pk, progress_data.key, chunk_size))
+    )
+
+    chord(tasks, interval=15)(_finish_delete_column.subtask([column_pk, progress_data.key]))
```

Comment:
It looks like we just have a single task we kick off, which in that case we can just use a [celery chain](https://docs.celeryproject.org/en/stable/userguide/canvas.html#chains), e.g.
```python
chain(
    _generate_tasks_to_delete_organization_column.subtask(...),
    _finish_delete_column.subtask(...)
).apply_async()
```